experimenter/experimenter/experiments/models.py

```diff
@@ -2116,6 +2116,7 @@ class NimbusVersionedSchema(models.Model):
     # Desktop-only
     set_pref_vars = models.JSONField[dict[str, str]](null=False, default=dict)
     is_early_startup = models.BooleanField(null=False, default=False)
+    has_remote_schema = models.BooleanField(null=False, default=False)
```

Comment:
Can we rename this something like `is_trivial_schema`, `is_autogenerated_schema` or something? It's not immediately clear what "remote" means.

Reply:
Oh I tried to match the naming of the other places it was referred to:

https://github.com/mozilla/experimenter/blob/511530effcb23e03948d1ed746d16e7dd5454b45/experimenter/experimenter/features/__init__.py#L43